crates/bevy_ecs_compile_fail_tests/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bevy_ecs_compile_fail_tests"
-version = "0.8.0-dev"
+version = "0.1.0"
```

Comment:
What's the motivation behind this change?

Reply:
This package is not part of the workspace, so its version will not get bumped and will stay forever at 0.8.0-dev, which is bad IMO. 0.1.0 is the initial version used by `cargo new` and is also consistent with our other internal-only crates, like those in `tools`.

Reply:
Sounds good.